auth_oauth_environment/models/auth_oauth_provider.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import json
+from lxml import etree
+
+from odoo import fields, models, api, _
+from odoo.addons.server_environment import serv_config
+
+
+class AuthOAuthProvider(models.Model):
+
+    _inherit = 'auth.oauth.provider'
```

Comment:
wouldn't be better to use the new mixin?
https://github.com/OCA/server-env/tree/11.0/server_environment#usage

Reply:
The mixin was introduced after this PR was done, so IMO we could merge it as is and it could be improved later on.